README.md

```diff
@@ -27,16 +27,8 @@ Once HWI's source has been downloaded with git clone, it and its dependencies ca
 poetry install
 ```
 
-Pip can also be used to install all of the dependencies (in virtualenv or system):
+Pip can also be used to install all of the dependencies (in virtualenv or system) required for operation and development. See `pyproject.toml` for all dependencies.
```

Comment:
It's probably helpful to also say that the dependencies for normal operation are those under `[tool.poetry.dependencies]` except for `pywin32-ctypes` which is Windows only (and of course Python itself which can't be installed with pip). To run the tests and build the binaries, you will need to install those under `[tool.poetry.dev-dependencies]` too.

Reply:
trying not to make this too verbose, hopefully that windows one will throw a legible error.

Reply:
made it a bit more verbose.